xarm_description/urdf/xarm6/xarm6.urdf.xacro

```diff
@@ -105,7 +105,7 @@
     <joint name="${prefix}joint2" type="revolute">
       <parent link="${prefix}link1"/>
       <child link="${prefix}link2"/>
-      <!-- <origin xyz="0 0 0" rpy="-1.5708 0 0" /> -->
+      <origin xyz="0 0 0" rpy="-1.5708 -0.5 0" />
```

Comment:
@NikolaRaicevic2001 : is this necessary?

Reply:
Yes, otherwise it will hit the table once it spawns. This raises the arm and changes the start configuration.